### ISSUE TITLE
Subtitle found when scanning within folder but not from the parentfolder for the same movie

### ISSUE THREAD

**Issue by Wesleyl89** [Diaoul/subliminal#731]
I have a video: 
```
/volume2/Movies/Jack Reacher- Never Go Back (2016)/Jack.Reacher.Never.Go.Back.2016.1080p.WEBDL.AC3.x264-FGT.mkv
```
When I try to find a subtitle from within this folder `/volume2/Movies/Jack Reacher- Never Go Back (2016)` it finds matching subtitles with 90+ scores.

But when I try to do the same but from within the parent directory `/volume2/Movies` it does find subtitles for that movie but only with scores around 60.

In the debug log from within the Jack Reacher folder it says the following that could have something to do with it:
```
INFO:subliminal.core:Refining video with metadata
DEBUG:subliminal.refiners.metadata:Found video_codec h264
DEBUG:subliminal.refiners.metadata:Found audio_codec AC3
DEBUG:subliminal.refiners.metadata:Found embedded subtitle set([<Language [und]>])
INFO:subliminal.core:Refining video with omdb
DEBUG:subliminal.refiners.omdb:Found 1 results
DEBUG:subliminal.refiners.omdb:Found result with matching year
DEBUG:subliminal.refiners.omdb:Found movie {u'imdbID': u'tt3393786', u'Title': u'Jack Reacher: Never Go Back', u'Type': u'movie', u'Poster': u'https://images-na.ssl-images-amazon.com/images/M/MV5BODQ3ODQ3NDI4NV5BMl5BanBnXkFtZTgwMDY1Mzk5OTE@._V1_SX300.jpg', u'Year': u'2016'}
```
In the debug log from the parent folder this:
```
INFO:subliminal.core:Refining video with metadata
DEBUG:subliminal.refiners.metadata:Found video_codec h264
DEBUG:subliminal.refiners.metadata:Found audio_codec AC3
DEBUG:subliminal.refiners.metadata:Found embedded subtitle set([<Language [und]>])
INFO:subliminal.core:Refining video with omdb
DEBUG:subliminal.refiners.omdb:Found 1 results
WARNING:subliminal.refiners.omdb:No matching movie found
```

debug log from `/volume2/Movies/Jack Reacher- Never Go Back (2016)`: http://pastebin.com/4MUWyezT
debug log from `/volume2/Movies` http://pastebin.com/3yKagKAM

```
root@LINNAS:/volume1/.@root # pip show subliminal
Name: subliminal
Version: 2.0.5
```

edit: I did some further testing and apparently it doesn't play well when the directory has a release year in the name like `Jack Reacher- Never Go Back (2016)`. Changing it to `Jack Reacher- Never Go Back` it works like it should.

**Comment by Diaoul** [comment]
It may be related to guessit, I'll try to reproduce to further investigate this.

**Comment by nateshmbhat** [comment]
Yes , i have been experiencing the same problem .

**Comment by ratoaq2** [comment]
This seems an old guessit issue that was fixed several months ago. Which guessit version are you using?

**Comment by fernandog** [comment]
@ratoaq2 with GuessIt 2.1.4:

With year in the folder it detects the second part of the title as an alternative_title

Year in folder:
```
guessit "/volume2/Movies/Jack Reacher- Never Go Back (2016)/Jack.Reacher.Never.Go.Back.2016.1080p.WEBDL.AC3.x264-FGT.mkv"
For: /volume2/Movies/Jack Reacher- Never Go Back (2016)/Jack.Reacher.Never.Go.Back.2016.1080p.WEBDL.AC3.x264-FGT.mkv
GuessIt found: {
    "title": "Jack Reacher",
    "alternative_title": "Never Go Back",
    "year": 2016,
    "screen_size": "1080p",
    "format": "WEB-DL",
    "audio_codec": "AC3",
    "video_codec": "h264",
    "release_group": "FGT",
    "container": "mkv",
    "mimetype": "video/x-matroska",
    "type": "movie"
}
```

Year not in folder:
```
guessit "/volume2/Movies/Jack Reacher- Never Go Back/Jack.Reacher.Never.Go.Back.2016.1080p.WEBDL.AC3.x264-FGT.mkv"
For: /volume2/Movies/Jack Reacher- Never Go Back/Jack.Reacher.Never.Go.Back.2016.1080p.WEBDL.AC3.x264-FGT.mkv
GuessIt found: {
    "title": "Jack Reacher Never Go Back",
    "year": 2016,
    "screen_size": "1080p",
    "format": "WEB-DL",
    "audio_codec": "AC3",
    "video_codec": "h264",
    "release_group": "FGT",
    "container": "mkv",
    "mimetype": "video/x-matroska",
    "type": "movie"
}
```

**Comment by ratoaq2** [comment]
Fixed by 982991de3daaac87e4f05534f227f7c4b111407f
